### PR TITLE
Applying terraform format to all files

### PR DESCRIPTION
**Pull request by Lakhtenkov-iv**

### CODE REVIEW

**Comment by Lakhtenkov-iv** [comment]
Hi @rarsan, @npredey.
I want to contribute feature for KMS encrypted tokens and token from Secretes.
The branch is huge and it is here: https://github.com/Lakhtenkov-iv/terraform-splunk-log-export/pull/4
In order to simplify PR review process I spitted it in to 3 PRs:
1. Fix terraform format
2.  Add terraform-docs support
3. Actual KMS and Secrets feature

Is PR is number 1 from the list. Could you please review and merge.

**Comment by rarsan** [comment]
Thank you @Lakhtenkov-iv for contributing this. I see you already signed the CLA.

I appreciate you breaking this down into 3 PRs. I will review all by Mon or Tue. Thanks.

File: main.tf [hunk 1]
```diff
@@ -44,29 +35,27 @@ locals {
   dataflow_temporary_gcs_bucket_path = "tmp/"
 
   dataflow_splunk_template_gcs_path = "gs://dataflow-templates/${var.dataflow_template_version}/Cloud_PubSub_to_Splunk"
-  dataflow_pubsub_template_gcs_path = "gs://dataflow-templates/${var.dataflow_template_version}/Cloud_PubSub_to_Cloud_PubSub"
```
Comment (rarsan) [inline main.tf:47]: This variable is used in the replay pipeline included in `replay.tf`. The replay pipeline is only deployed to recover undelivered messages as described in this [solution guide](https://cloud.google.com/architecture/deploying-production-ready-log-exports-to-splunk-using-dataflow#handling_delivery_failures). Therefore it's commented out by default in the `replay.tf` file. There are probably better ways to do this, but for now we probably add lint warning ignore around this.

File: main.tf [hunk 2]
```diff
@@ -44,29 +35,27 @@ locals {
...
   # otherwise, use Compute Engine default service account
   dataflow_worker_service_account = ((var.dataflow_worker_service_account != "")
     ? "${var.dataflow_worker_service_account}@${var.project}.iam.gserviceaccount.com"
-    : "${data.google_project.project.number}-compute@developer.gserviceaccount.com")
+  : "${data.google_project.project.number}-compute@developer.gserviceaccount.com")
 
-  subnet_name = coalesce(var.subnet, "${var.network}-${var.region}")
+  subnet_name           = coalesce(var.subnet, "${var.network}-${var.region}")
   project_log_sink_name = "${var.dataflow_job_name}-project-log-sink"
-  organization_log_sink_name = "${var.dataflow_job_name}-organization-log-sink"
```
Comment (rarsan) [inline main.tf:57]: Same with this variable which is used in a commented out resource `organization_log_sink`. Let's add tflint-ignore for now? We can add a terraform variable in subsequent PR to conditionally control creation of the org log sink (in addition to new organization_id TF variable).

File: versions.tf [hunk 3]
```diff
@@ -0,0 +1,13 @@
+terraform {
```
Comment (rarsan) [inline versions.tf:1]: Thanks for adding the providers and versions files.
I don't think I've personally tested these older provider versions though.
Has this been tested with these versions? For example, `random` version below is about 5 yrs old.

**Review by rarsan — CHANGES_REQUESTED** [review]
Thanks for your patience. Looks good! Some minor comments below. Please see note re 2 required variables that were accidentally removed.

With respect to the workflow yaml file, I appreciate you adding that. Do we need to add any usage instructions?

File: main.tf (hunk 1 above)
Comment (Lakhtenkov-iv) [reply main.tf:47]: Sorry I didn't get the point that I need to uncomment the code to deploy. I have brought back variable.

File: main.tf (hunk 2 above)
Comment (Lakhtenkov-iv) [reply main.tf:57]: Fixed as well.

File: versions.tf (hunk 3 above)
Comment (Lakhtenkov-iv) [reply versions.tf:1]: Yes, you are right we need to bump up all the 3: `terraform`, `google` and `random` provider. I will test. Please don't merge

Comment (Lakhtenkov-iv) [reply versions.tf:1]: Now it is good, I've tested with minimal versions that is defined.

**Comment by Lakhtenkov-iv** [comment]
Hi @rarsan. Any feedback on this PR?

**Comment by rarsan** [comment]
Thanks for the fixes! PR merged.

**Pull request merged**